---
title: "'Finally' può essere usato solo una volta in un'istruzione 'Try'"
ms.date: 07/20/2015
f1_keywords:
- vbc30381
- bc30381
helpviewer_keywords:
- BC30381
ms.assetid: 4fa1d5fa-c54a-4f8c-9d66-9dbcc38c53bf
ms.openlocfilehash: 61328bd808e071670b72350c893960df78a96b66
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 06/04/2020
ms.locfileid: "84403745"
---
# <a name="finally-can-only-appear-once-in-a-try-statement"></a>'Finally' può essere usato solo una volta in un'istruzione 'Try'
`Finally` viene usato per completare un blocco `Try...Catch...Finally` , quindi può essere presente solo una volta alla fine del blocco.  
  
 **ID errore:** BC30381  
  
## <a name="to-correct-this-error"></a>Per correggere l'errore  
  
1. Individuare e rimuovere l'oggetto `Finally`ridondante.  
  
## <a name="see-also"></a>Vedere anche

- [Istruzione Try...Catch...Finally](../language-reference/statements/try-catch-finally-statement.md)
